src/App.css

```diff
@@ -2,7 +2,7 @@
   max-width: 1280px;
   margin: 0 auto;
   padding: 2rem;
-  text-align: center;
+  /* text-align: center; */
```

Comment:
no need to merge commented out css if it's not serving a purpose

Reply:
okay I'll delete the commented out comment.